templates/default/rt/rtqueueadd.html

```diff
@@ -176,6 +176,26 @@ <H1>{$layout.pagetitle}</H1>
 									<TEXTAREA name="queue[resolveticketbody]" cols="60" rows="10" {tip text="Body of mail which is sent to customer when the ticket is resolved" trigger="resolveticketbody"}>{$queue.resolveticketbody}</TEXTAREA>
 								</TD>
 							</TR>
+							<TR>
```

Comment:
Tutaj proponuję dla całej tabeli wydzielić **COLGROUP** i do tego używać COLSPAN - coś mi się wydaje, że użycie po prostu szerokość 100% spowoduje problem w różnych konfiguracjach przeglądarek.

Reply:
A tabele zagnieżdżone?

Reply:
Poszedł commit nie ma już znaczników width w stylach poza COLGROUP.

Reply:
Jeszcze pozostałe pliki html zaraz to dorobię.